build.gradle

```diff
@@ -236,29 +236,33 @@ allprojects {
         google()
         mavenCentral()
         // org.jetbrains.trove4j:trove4j:20160824.
-        maven { url 'https://plugins.gradle.org/m2/' }
+        maven {
+            url 'https://plugins.gradle.org/m2/'
+        }
         maven {
             url 'https://trouterpublicpackages.z13.web.core.windows.net'
         }
     }
 
     ext {
+        androidxTestVersion = "1.3.0"
         annotationsVersion = "1.1.0"
         appCompatVersion = "1.2.0"
         archCoreTestVersion = "2.1.0"
+        azureCoreVersion = "1.0.0-beta.4"
         concurrentFuturesVersion = "1.0.0"
-        jUnitVersion = "5.7.0"
-        androidxTestVersion = "1.3.0"
+        dataproviderVersion =  "2.6"
         jacksonVersion = "2.11.2"
+        jUnitVersion = "5.7.0"
         lifecycleVersion = "2.2.0"
+        nimbusJoseJwtVersion = "9.1.2"
```

Comment:
can we make it explicit that this version is allowed only for unit testing? just to avoid taking an api /implementation dependency by mistake. something like "nimbusJoseJwtTestVersion"